### PR TITLE
create WenDocs Linker

### PR DESCRIPTION
**Pull request by JulioWendocs**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/WenDocs Linker/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+                  "description": "For API service users, please use 'API Service'"
+                }
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "response body",
```
Comment (sriyen-msft) [inline certified-connectors/WenDocs Linker/apiDefinition.swagger.json:80]: @JulioWendocs Can you please provide valid descriptions for all the response codes.

File: certified-connectors/WenDocs Linker/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+                  "type": "string",
+                  "description": "documentName"
+                },
+                "document": {
+                  "type": "string",
+                  "description": "document"
+                },
+                "errorMessage": {
+                  "type": "string",
+                  "description": "errorMessage"
```
Comment (sriyen-msft) [inline certified-connectors/WenDocs Linker/apiDefinition.swagger.json:94]: @JulioWendocs The description values for properties should be free from grammatical and spelling errors. Please provide valid description values for all the properties.

Documentation - [Parameter descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/WenDocs Linker/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#92D050",
+    "capabilities": ["actions"],
+    "publisher": "WenDocs Ltd",
+    "stackOwner": "Beijing Huawen Yundao Technology Co., Ltd"
```
Comment (sriyen-msft) [inline certified-connectors/WenDocs Linker/apiProperties.json:34]: @JulioWendocs Please provide us any document which shows relationship b/w the stackOwner and publisher ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by JulioWendocs** [comment]
Hi Sriyen:
First, I have updated the json file on my github, please let me know how should I update the pull request.


"Beijing Huawen Yundao Technology Co., Ltd" is the english name of "北京华文云道科技有限公司" and our company is also a MPN. "Wendocs" is our url "www.wendocs.com" and product name. We have published a word add-in "WenDocs Template Designer" in office store. Is this ok to prove the relationship?





Thanks，







 
 在2021年07月13 09时55分，"Srikanth ***@***.***>写道：


@sriyen-msft requested changes on this pull request.
In certified-connectors/WenDocs Linker/apiDefinition.swagger.json:
> +                "country": {
+                  "type": "string",
+                  "description": "supported country code of error message: US, CN"
+                },
+                "clientType": {
+                  "type": "string",
+                  "description": "For API service users, please use 'API Service'"
+                }
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "response body",@JulioWendocs Can you please provide valid descriptions for all the response codes.
In certified-connectors/WenDocs Linker/apiDefinition.swagger.json:
> +          "200": {
+            "description": "response body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "documentName": {
+                  "type": "string",
+                  "description": "documentName"
+                },
+                "document": {
+                  "type": "string",
+                  "description": "document"
+                },
+                "errorMessage": {
+                  "type": "string",
+                  "description": "errorMessage"@JulioWendocs The description values for properties should be free from grammatical and spelling errors. Please provide valid description values for all the properties.
Documentation - Parameter descriptions
In certified-connectors/WenDocs Linker/apiProperties.json:
> +        "uiDefinition": {
+          "displayName": "API_Secret",
+          "description": "the API_Secret for access publish api service",
+          "tooltip": "provide the API_Secret obtained from WenDocs Corporation[www.wendocs.com]",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#92D050",
+    "capabilities": ["actions"],
+    "publisher": "WenDocs Ltd",
+    "stackOwner": "Beijing Huawen Yundao Technology Co., Ltd"@JulioWendocs Please provide us any document which shows relationship b/w the stackOwner and publisher ?

File: certified-connectors/WenDocs Linker/apiDefinition.swagger.json (hunk 2 above)
Comment (JulioWendocs) [reply certified-connectors/WenDocs Linker/apiDefinition.swagger.json]: apiDefinition.swagger.json has been updated to give valid description.

**Comment by sriyen-msft** [comment]
> Hi Sriyen: First, I have updated the json file on my github, please let me know how should I update the pull request. "Beijing Huawen Yundao Technology Co., Ltd" is the english name of "北京华文云道科技有限公司" and our company is also a MPN. "Wendocs" is our url "[www.wendocs.com](http://www.wendocs.com)" and product name. We have published a word add-in "WenDocs Template Designer" in office store. Is this ok to prove the relationship? Thanks， 在2021年07月13 09时55分，"Srikanth ***@***.***>写道： @sriyen-msft requested changes on this pull request. In certified-connectors/WenDocs Linker/apiDefinition.swagger.json:
> + "country": {
> + "type": "string", + "description": "supported country code of error message: US, CN" + }, + "clientType": { + "type": "string", + "description": "For API service users, please use 'API Service'" + } + } + }, + "required": true + } + ], + "responses": { + "200": { + "description": "response body",@JulioWendocs Can you please provide valid descriptions for all the response codes. In certified-connectors/WenDocs Linker/apiDefinition.swagger.json:
> + "200": {
> + "description": "response body", + "schema": { + "type": "object", + "properties": { + "documentName": { + "type": "string", + "description": "documentName" + }, + "document": { + "type": "string", + "description": "document" + }, + "errorMessage": { + "type": "string", + "description": "errorMessage"@JulioWendocs The description values for properties should be free from grammatical and spelling errors. Please provide valid description values for all the properties. Documentation - Parameter descriptions In certified-connectors/WenDocs Linker/apiProperties.json:
> + "uiDefinition": {
> + "displayName": "API_Secret", + "description": "the API_Secret for access publish api service", + "tooltip": "provide the API_Secret obtained from WenDocs Corporation[[www.wendocs.com](http://www.wendocs.com)]", + "constraints": { + "tabIndex": 3, + "clearText": false, + "required": "true" + } + } + } + }, + "iconBrandColor": "#92D050", + "capabilities": ["actions"], + "publisher": "WenDocs Ltd", + "stackOwner": "Beijing Huawen Yundao Technology Co., Ltd"@JulioWendocs Please provide us any document which shows relationship b/w the stackOwner and publisher ? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

Thank you so much for clarification

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**